spaghetto/bin/src/Main.purs

```diff
@@ -2,29 +2,28 @@ module Main (main) where
 
 import Spago.Prelude
 
-import ArgParse.Basic (ArgParser)
-import ArgParse.Basic as ArgParser
 import Data.Array as Array
 import Data.Array.NonEmpty as NonEmptyArray
 import Data.Codec.Argonaut.Common as CA.Common
+import Data.Foldable (fold)
 import Data.JSDate as JSDate
 import Data.List as List
 import Data.Map as Map
 import Data.Set.NonEmpty (NonEmptySet)
 import Data.String as String
 import Effect.Aff as Aff
-import Effect.Class.Console as Console
 import Effect.Ref as Ref
 import Node.FS.Stats (Stats(..))
 import Node.Path as Path
 import Node.Process as Process
+import Options.Applicative (CommandFields, Mod, Parser, ParserPrefs(..), command, customExecParser, defaultPrefs, helper, hsubparser, info, progDesc)
```

Comment:
Same note about the qualified imports here

Reply:
https://github.com/purescript/spago/pull/980/commits/bf02067c43dab3d244e2761f8149efce6b03cb9d